scylla/Cargo.toml

```diff
@@ -2,10 +2,10 @@
 name = "scylla"
 version = "0.0.1"
 edition = "2018"
-description = "Async CQL driver for Rust, optimized for Scylla."
-repository = "https://github.com/psarna/scylla-rust-driver"
+description = "Async CQL driver for Rust, optimized for Scylla, fully compatible with Apache Cassandra™"
```

Comment:
"Apache Cassandra" is a trademark, but "Scylla" is not? How does it work?

Reply:
Apache Cassandra people care about trademarks appearing in various contributions, while we don't care if Scylla is trademarked in our own code (: